blocks/identity-block/components/Button/styles.scss

```diff
@@ -24,7 +28,7 @@
 }
```

Comment:
icon container requires `display: flex` to be set to align the icon and text correctly

Reply:
updated I think in the right place. seems to be already using flex? 

![Screen Shot 2021-09-02 at 10 26 29](https://user-images.githubusercontent.com/5950956/131872556-e7f70bd9-2eb8-4eec-bf37-2bfd1f2470af.png)